zproject/computed_settings.py

```diff
@@ -905,12 +905,12 @@ def zulip_path(path: str) -> str:
         },
         'zulip.auth': {
             'level': 'DEBUG',
-            'handlers': DEFAULT_ZULIP_HANDLERS + ['auth_file'],
+            'handlers': ['file', 'auth_file', 'errors_file'],
             'propagate': False,
         },
         'zulip.ldap': {
             'level': 'DEBUG',
-            'handlers': ['console', 'ldap_file', 'errors_file'],
+            'handlers': ['file', 'ldap_file', 'errors_file'],
             'propagate': False,
```

Comment:
Should `django_auth_ldap` and `pika` have the same change?

I'm a little skeptical of this change in general -- things we remove the `console` logger from won't be logged to stdout, which I think is important for some of these in `manage.py sync_ldap_user_data` cases.  

The reason many similar log files don't have `console` listed is that we don't want to print output to users.

Reply:
Okay, I agree with that reasoning to keep console on this logger. I removed this change, so only the `zulip.auth` change is left, and those loggers were the cause of most of the console spam on `test-backend` runs.

I don't know about pika and django_auth_ldap, I'm not aware of the reasons they were configured this way, so that's why I didn't touch them.

Reply:
Hmm, what's the argument for why it's correct to remove `zulip.auth`?

I believe making this change will make those errors no longer appear in the `run-dev.py` console, which seems like a bug to me.

Reply:
Well the argument was that it was spamming `test-backend` output and confusing folks in czo topic linked above, but on second thought, that console output from `run-dev.py` is super useful when debugging authentication methods, so let's keep the output indeed. I'll close the PR.